aci/resource_aci_cloudsubnet.go

```diff
@@ -79,30 +81,42 @@ func resourceAciCloudSubnet() *schema.Resource {
 			},
 
 			"zone": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				Computed: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "Only applicable to the AWS vendor",
 			},
 
 			"relation_cloud_rs_subnet_to_flow_log": &schema.Schema{
 				Type: schema.TypeString,
 
 				Optional: true,
 			},
+			"subnet_group_label": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Only applicable to the GCP vendor",
+			},
 		}),
 	}
 }
-func getRemoteCloudSubnet(client *client.Client, dn string) (*models.CloudSubnet, error) {
-	cloudSubnetCont, err := client.Get(dn)
+func getRemoteCloudSubnet(client *client.Client, dn string, d *schema.ResourceData) (*models.CloudSubnet, error) {
+	dnUrl := fmt.Sprintf("%s/%s.json?rsp-subtree=children", client.MOURL, dn)
+
+	cloudSubnetCont, err := client.GetViaURL(dnUrl)
 	if err != nil {
 		return nil, err
 	}
 
 	cloudSubnet := models.CloudSubnetFromContainer(cloudSubnetCont)
 
+	// To set cloudSubnet relational attributes value
+	setCloudSubnetRelationalAttributes(cloudSubnetCont, d)
+
 	if cloudSubnet.DistinguishedName == "" {
-		return nil, fmt.Errorf("CloudSubnet %s not found", cloudSubnet.DistinguishedName)
+		return nil, fmt.Errorf("CloudSubnet %s not found", dn)
```

Comment:
CloudSubnet -> Cloud Subnet

Reply:
It would help while debugging the error messages with the original class name.